native/screens/Applicants/Applicants.jsx

```diff
@@ -74,7 +68,7 @@ const RoleList = ({ applicants, navigation, role, currProject }) => {
     return (
       <View>
```

Comment:
The container for each RoleList should have at least a 40px between each RoleList component, drew a line on figma to measure the distance